components/board/genericlinux/pin_types.go

```diff
@@ -41,3 +33,61 @@ type NoBoardFoundError struct {
 func (err NoBoardFoundError) Error() string {
 	return fmt.Sprintf("could not determine %q model", err.modelName)
 }
+
+// PinDefinition describes a gpio pin on a linux board.
+type PinDefinition struct {
+	Name            string `json:"name"`
+	Ngpio           int    `json:"ngpio"` // this is the ngpio number of the chip the pin is attached to
```

Comment:
I'm open to disagreement, but I'd remove the "this is" at the beginning of the comment. Starting with "the ngpio number" seems more straightforward, IMO.